src/tools/pontoon/translation_workspace.md

```diff
@@ -192,6 +193,14 @@ Icons to the right indicate the [status](#translation-status) of each translatio
 
 In the screenshot above, the first item is the approved translation (green checkmark), while the other two are rejected suggestions.
 
+#### Status banners
+
+Status banners reflect the permissions of the user within the respective locale and project. Each translator’s banner is displayed at the bottom of their profile picture.
+
+![List of three possible status banners](../../assets/images/pontoon/translation_workspace/status_banners.png "Screenshot of list of suggestions displaying 3/5 possible banners for users")
+
+Namely, team managers will get the `MNGR` tag, project managers get the `PM` tag, translators get the `TRNSL` tag, and those with site-wide admin permissions receive the `ADMIN` tag. Users who have joined within the last three months will get the `NEW USER` tag for their banner. Status banners also appear in translation comments (see below).
```

Comment:
This might be clearer as a list, with banner label and explanation.

These are the available status banners:
* `MNGR` for team managers.
* etc.